modules/build/src/main/scala/scala/build/interactive/Interactive.scala

```diff
@@ -1,44 +1,79 @@
 package scala.build.interactive
-
-import scala.build.errors.BuildException
 import scala.build.options.BuildOptions
 import scala.io.StdIn.readLine
 
 case class Interactive(options: BuildOptions) {
```

Comment:
Should other values be read from `options` in the future? If not, maybe we could have a `trait Interactive`, without most implems, and a `InteractiveAsk` say, used when `options.internal.interactive.getOrElse(false)` is true, along with a `InteractiveNop`, whose methods would always return `None`. And we could create the `Interactive` instance in one place, and pass it around.

Reply:
I create `Interactive` instance in `BuildOptions`, but is some cases such as Update there is no available `BuildOptions `instance, so I created `Interactive` in `VerbosityOptionsUtil`.